aten/src/ATen/native/cuda/BatchLinearAlgebraLib.cu

```diff
@@ -826,6 +826,55 @@ Tensor& orgqr_helper_cusolver(Tensor& result, const Tensor& tau) {
   return result;
 }
 
+void apply_lu_cusolver_looped(Tensor& self, Tensor& pivots, Tensor& infos, bool get_pivots) {
```

Comment:
For other existing linear algebra function we have a non-documented convention that "apply_*" functions are for templated by dtype implementation calling LAPACK API routines, here the lambda function that is inside `AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES` is what "apply_" is called in other places.

Let's rename this to `lu_looped_cusolver`.

Reply:
I see. Will rename.